### PR TITLE
Added Google Analytics tags to these items

### PR DESCRIPTION
**Pull request by GK230**
Added Google Analytics tags to profile drop down, custodian team dashboard and customise DAR page

### CODE REVIEW

**Comment by kandaj** [comment]
@GK230 can you import the handleAnalytics function from handleAnalytics.js from dataAccessRequestCustomiseForm folder?

**Review by kandaj — DISMISSED** [review]

File: src/pages/dashboard/Components/DashboardNavAccordian/DashboardNavAccordion.js [hunk 1]
```diff
@@ -13,7 +16,7 @@ const DashboardNavItem = ({ activeKey, tabId, eventKey, onSelect, onClick, data
             <Accordion.Collapse eventKey={eventKey}>
                 <div>
                     {data.children.map(({ id, text }) => (
-                        <DashboardSubNavItem active={tabId === id} onClick={() => onClick(id)}>
+                        <DashboardSubNavItem active={tabId === id} onClick={() => handleAnalytics(`Clicked on ${id}`, text)}>
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dashboard/Components/DashboardNavAccordian/DashboardNavAccordion.js:19]: You've removed the onClick?

File: src/pages/dashboard/CustomiseDAR/CustomiseDAR.js [hunk 2]
```diff
@@ -178,10 +180,10 @@ const CustomiseDAR = ({ userState, publisherId, showConfirmPublishModal, setShow
                     </div>
                     <div className='tabsBackground mb-3'>
                         <Tabs className='dataAccessTabs gray700-13' activeKey={activeTab} onSelect={handleSelectTab}>
-                            <Tab eventKey='customisedataaccessrequests_guidance' title={t('tabs.presubmissionGuidance')}>
+                            <Tab eventKey='customisedataaccessrequests_guidance' title={t('tabs.presubmissionGuidance')} onClick={() => handleAnalytics('Clicked on Presubmission Guidance card', 'Presubmission Guidance' )}>
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dashboard/CustomiseDAR/CustomiseDAR.js:183]: Can you just do this in onSelect / handleSelectTab and use the dynamic key passed in

**Review by ThinkingMonkeyLtd — APPROVED** [review]

File: src/pages/dashboard/Components/DashboardNavAccordian/DashboardNavAccordion.js (hunk 1 above)
Comment (kandaj) [reply src/pages/dashboard/Components/DashboardNavAccordian/DashboardNavAccordion.js:19]: @ThinkingMonkeyLtd  good spoting

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by kandaj — APPROVED** [review]

**Pull request merged**